doc/getting_started/setup_dv.md

```diff
@@ -114,5 +115,5 @@ $ util/dvsim/dvsim.py hw/top_earlgrey/dv/top_earlgrey_sim_cfgs.hjson -i smoke --
 We can only run the failed test with `--fixed-seed=1` to reproduce the failure.
 Assume there is a failure in the `uart_smoke` test. Here is the run command to reproduce it.
 ```console
-$ util/dvsim/dvsim.py hw/ip/uart/dv/uart_sim_cfg.hjson -i uart_smoke --fixed-seed=1 [--waves]
+$ util/dvsim/dvsim.py hw/ip/uart/dv/uart_sim_cfg.hjson -i uart_smoke --fixed-seed=1 [--waves vcd]
```

Comment:
VCD format is huge (orders of magnitude sometimes) compared to others. Do we want this to be what we show in the example over say `fsdb` or `vpd`? @sriyerg can comment more

Reply:
VCD files are indeed huge, and I'd rather avoid any mention of vcd files. I was only trying to offer a command that works whichever simulator is being used. I could flesh out the description to describe explicitly which simulator and format combinations are available/supported?

Reply:
I think we can just leave this as `[--waves <format>]`, if we are already mentioning the list of supported formats above.